pallets/stateful-storage/src/rpc/src/tests/mod.rs

```diff
@@ -0,0 +1,136 @@
+mod rpc_mock;
+
+use super::*;
+use rpc_mock::*;
+
+use common_primitives::stateful_storage::*;
+use pallet_stateful_storage_runtime_api::StatefulStorageRuntimeApi;
+use sp_runtime::DispatchError;
+use std::sync::Arc;
+use substrate_test_runtime_client::runtime::Block;
+
+const SUCCESSFUL_SCHEMA_ID: u16 = 1;
+const SUCCESSFUL_MSA_ID: MessageSourceId = 1;
+const SUCCESSFUL_PAYLOAD: &[u8; 33] = b"{'body':827, 'val':'another val'}";
+const DUMMY_STATE_HASH: u32 = 32767;
+
+sp_api::mock_impl_runtime_apis! {
+	impl StatefulStorageRuntimeApi<Block> for TestRuntimeApi {
+		/// Retrieve the itemized storages for a particular msa and schema
+		fn get_paginated_storages(msa_id: MessageSourceId, schema_id: SchemaId) -> Result<Vec<PaginatedStorageResponse>, DispatchError> {
+			match (msa_id, schema_id) {
+				(SUCCESSFUL_MSA_ID, SUCCESSFUL_SCHEMA_ID) => Ok(vec![PaginatedStorageResponse::new(
+					0,
+					msa_id,
+					schema_id,
+					SUCCESSFUL_PAYLOAD.to_vec(),
+					DUMMY_STATE_HASH)]),
+				_ => Err(DispatchError::Other("some error")),
+			}
+		}
+
+		fn get_itemized_storages(msa_id: MessageSourceId, schema_id: SchemaId) -> Result<ItemizedStoragePageResponse, DispatchError> {
```

Comment:
```suggestion
		fn get_itemized_storage(msa_id: MessageSourceId, schema_id: SchemaId) -> Result<ItemizedStoragePageResponse, DispatchError> {
```

Reply:
Ignore me, apparently all the cool Substrate kids are using "storages" these days 🤷‍♀️